enterprise/models.py

```diff
@@ -524,17 +515,8 @@ class Meta:
         default=False,
     )
 
-    career_engagement_network_message = models.TextField(
-        blank=True,
-        null=True,
-        help_text=_(
-            'Message text shown on the learner portal dashboard for career engagement network.'
-        ),
-    )
-
     learner_portal_sidebar_content = models.TextField(
         blank=True,
-        null=True,
```

Comment:
why are we removing null from here

Reply:
Because we changed the signature of this field to pass the tests